pkgs/tools/system/thermald/default.nix

```
@@ -1,24 +1,47 @@
-{ stdenv, fetchFromGitHub, autoconf, automake, libtool
-, pkgconfig, dbus, dbus-glib, libxml2, autoconf-archive }:
+{ stdenv,
+ fetchFromGitHub,
+ autoconf,
+ automake,
+ libtool ,
+ pkgconfig,
+ dbus,
+ dbus-glib,
+ libxml2,
+ autoconf-archive,
+ gtk-doc,
+ lzma,
+ upower,
+ libevdev }:
```

```suggestion
 , autoconf
 , automake
 , libtool
 pkgconfig,
 dbus,
 dbus-glib,
 libxml2,
 autoconf-archive,
 gtk-doc,
 lzma,
 upower,
 libevdev 
 }:
```
Commas typically go before package, and ``}:`` ends on its own line.
Try running ``nix-shell -p nixpkgs-fmt --run "path/to/this/file.nix"``